src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState.cs

```diff
@@ -1497,13 +1557,21 @@ public SolutionCompilationState RemoveAnalyzerConfigDocuments(ImmutableArray<Doc
         IEnumerable<(ProjectId projectId, ImmutableArray<TDocumentState> newDocumentStates)> projectIdAndNewDocuments,
         Func<ProjectState, ImmutableArray<TDocumentState>, TranslationAction> addDocumentsToProjectState)
         where TDocumentState : TextDocumentState
+    {
+        return UpdateDocumentsInMultipleProjects(projectIdAndNewDocuments, addDocumentsToProjectState);
+    }
+
+    private SolutionCompilationState UpdateDocumentsInMultipleProjects<TDocumentState>(
```

Comment:
>UpdateDocumentsInMultipleProjects [](http://example.com/codeflow?start=37&length=33)

Isn't this essentially a rename of AddDocumentsToMultipleProjects? If so, why keep the old one?

Reply:
yes.  removed hte old one.